119.md

```diff
@@ -0,0 +1,18 @@
+# NIP-119: AND Operator in Filters
+
+Enable `AND` within a single tag filter by using an `&` modifier in filters for indexable tags.
+
+```
+filters: {
+  "kinds": [1],
+  "&t": ["meme", "cat"],
+  "#t": ["black", "white"]
+}
+// returns kind `1` events with `t` tags that have both "meme" and "cat" that have the tag "black" or "white"
+```
+
+## Rules
+
+- `AND` **MUST** take precedence over `OR`
+- Tags used in `AND` **SHOULD NOT** be used in standard `OR` tags [`#`]
+- Any tag used in `AND` **SHOULD** be ignored in `OR` 
```

Comment:
Can you elaborate on these rules? They say

```Tags used in `AND` **SHOULD NOT** be used in standard `OR` tags```

and yet in your example filter you specifically use the ```t``` tag in both AND and OR filters.

Reply:
Got it, second point is asking clients "please don't do this", third is telling relays "if they do it, ignore it"